tests/test_core.py

```diff
@@ -76,6 +76,7 @@ def blank_dagobah():
                                              os.path.dirname(__file__)))
     dagobah = Dagobah(backend, ssh_config=os.path.join(location,
                                                        "test_ssh_config"))
+    dagobah.JIJ_DELIM = "|"
```

Comment:
This is merely for test purposes, to make reading the results easier if something goes wrong.